### PR TITLE
Neo.VM.3.0.0-CI00958

### PR DESCRIPTION
**Pull request by Tommo-L**
Close https://github.com/neo-project/neo-modules/issues/270

### CODE REVIEW

File: tests/Neo.Network.RPC.Tests/RpcTestCases.json [hunk 1]
```diff
@@ -427,7 +427,7 @@
                   }
                 ],
                 "offset": 0,
-                "returnType": "Signature"
+                "returnType": "Any"
```
Comment (Tommo-L) [inline tests/Neo.Network.RPC.Tests/RpcTestCases.json:430]: @shargon  @erikzhang  Do you know why did it change here? It seems  we haven't changed `event`.

Comment (shargon) [reply tests/Neo.Network.RPC.Tests/RpcTestCases.json:430]: I am not sure...

Comment (erikzhang) [reply tests/Neo.Network.RPC.Tests/RpcTestCases.json:430]: Why `event` has a return type?

Comment (Tommo-L) [reply tests/Neo.Network.RPC.Tests/RpcTestCases.json:430]: It should be removed.

Comment (erikzhang) [reply tests/Neo.Network.RPC.Tests/RpcTestCases.json:430]: See https://github.com/neo-project/neo/pull/1733

**Pull request merged**